### DISCUSSION TITLE
Side Panel Overflow with Long Repo Names

### DISCUSSION THREAD

**Discussion by kent-daniel** [yhype/yhype-feedback#33] [Bug]
Hey, I noticed that if a repo name is too long, it tends to overflow in the side panel, which looks a bit messy. It would be great to add some kind of [word wrap](https://www.w3schools.com/cssref/css3_pr_word-wrap.php) or text wrapping so that everything stays neat and readable. <br/>
![image](https://github.com/user-attachments/assets/7f79bf68-dba7-4f09-9dd9-ff5a972a00a6)

Thanks!

**Comment by antonkomarev** [comment]
@kent-daniel 
You can try new interface, it includes dark theme and has mobile support now!

<img width="1511" height="829" alt="image" src="https://github.com/user-attachments/assets/22065a8d-688e-4184-b03e-00bfb2766bec" />
